tools/walletextension/accountmanager/account_manager.go

```diff
@@ -318,14 +323,62 @@ func (r *RPCRequest) Clone() *RPCRequest {
 
 // convertToCallMsg converts the interface to a *ethereum.CallMsg
 func convertToCallMsg(callMsgInterface interface{}) (*ethereum.CallMsg, error) {
-	callMsgBytes, err := json.Marshal(callMsgInterface)
-	if err != nil {
-		return nil, fmt.Errorf("unable to marshall callMsg - %w", err)
-	}
-	var callMsg ethereum.CallMsg
-	err = json.Unmarshal(callMsgBytes, &callMsg)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse callMsg - %w", err)
+	// geth lowercases the field name and uses the last seen value
```

Comment:
Taking ideas on where it should live - I thought of go/common/marshallers ? but it doesnt feel 100% right